src/Tribe/Tickets.php

```diff
@@ -882,9 +882,8 @@ public function ajax_refresh_settings() {
 			$ticket->description      = isset( $data['ticket_description'] ) ? esc_html( $data['ticket_description'] ) : null;
 			$ticket->price            = ! empty( $data['ticket_price'] ) ? filter_var( trim( $data['ticket_price'] ), FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION | FILTER_FLAG_ALLOW_THOUSAND ) : 0;
 			$ticket->purchase_limit   = isset( $data['ticket_purchase_limit'] ) ? absint( $data['ticket_purchase_limit'] ) : apply_filters( 'tribe_tickets_default_purchase_limit', 0, $ticket->ID );
-
-			$show_description = isset( $data['ticket_show_description'] ) ? absint( $data['ticket_show_description'] ) : 0;
-			$ticket->show_description = $ticket->show_description();
+			$ticket->show_description = isset( $data['ticket_show_description'] ) ? 'yes' : 'no';
```

Comment:
everywhere else this seems to use `1` or `0` (maybe `true`/`false` - why introduce `yes` and `no` here?

Reply:
why not call `$ticket->show_description();` here like in Tickets Plus?

Reply:
Because on this case it is where we initially start to save the data that is sent to ET+

Reply:
For consistency.  I used `yes`/`no` like WooCommerce uses.